Add turnaround time bar chart

bronzehedwick opened the pull request:
First pass at a chart showing turnaround times.

Operate on completed requests, showing how long it took to complete.
Since there are more than a hundred unique days-to-complete, putting
them all in a chart is unreadable, at least with my struggling
understanding of nivo.

To compensate, and to stop fighting the library, I manually grouped the
items coarsely.

## Screenshot
<img width="780" alt="Screen Shot 2021-01-09 at 9 18 24 PM" src="https://user-images.githubusercontent.com/637174/104112699-4503af80-52c0-11eb-8b27-ff1e6ec47e78.png">

bronzehedwick committed:
Add turnaround time bar chart

First pass at a chart showing turnaround times.

Operate on completed requests, showing how long it took to complete.
Since there are more than a hundred unique days-to-complete, putting
them all in a chart is unreadable, at least with my struggling
understanding of nivo.

To compensate, and to stop fighting the library, I manually grouped the
items coarsely.

aniagaska commented:
Is there a way to show the range between the quickest turnaround time or the longest turnaround time? Or are the lowest all around 10 days, and the longest around 150? Maybe a scatterplot is a better way to show this data, because it would give you the distribution of all the times?

bronzehedwick commented:
@aniagaska The range of days to completion is from 1 to 207 currently. Happy to discuss alternative visualizations. Here's the full list:

```
1
2
3
4
7
9
10
11
12
15
16
19
20
21
22
24
25
26
30
31
32
33
34
35
36
39
40
41
42
44
46
47
50
53
54
55
57
60
61
63
65
67
68
69
73
74
75
78
82
83
85
88
89
91
92
94
97
99
101
102
106
107
110
111
113
118
119
126
127
132
134
137
140
143
148
154
155
159
166
167
168
170
172
178
179
182
186
187
188
194
195
197
202
204
205
207
```

aniagaska commented:
I'm playing around with the data here https://docs.google.com/spreadsheets/d/1WTLD5O6j-3I0x085lYbk9t5eawDUt8vV8FNFNMYczFk/edit?usp=sharing the scatter plot doesn't look all that exciting right now

bronzehedwick commented:
@aniagaska I updated this PR with the numerical labels (screenshot below).

I added a new task for the dynamic grouping, since it can wait and needs more thought on my part.

<img width="728" alt="Screen Shot 2021-01-12 at 9 29 16 PM" src="https://user-images.githubusercontent.com/637174/104398779-644b4880-551d-11eb-96ad-3e3d1d610770.png">

If it looks good go ahead and merge :)
